SepulchureSaga/03SepulchuresRise.cs

```diff
@@ -0,0 +1,209 @@
+
+        //Disarm the Knights 6379
+        Story.KillQuest(6379, "ebonslate", "Ebonslate Knight");
+
+        //Bruise the Bruiser 6380
+        Story.KillQuest(6380, "ebonslate", "Ebonslate Bruiser");
+
+        //Take Down Dethrix 6381
+        if (Story.QuestProgression(6381))
+        {
+            Core.EnsureAccept(6381);
+            Core.Join("ebonslate", "r11", "Left");
+            Bot.Player.Attack("Dethrix");
+            Bot.Sleep(10000);
+        }
+
+        //Please Fix Me 6382
+        Story.KillQuest(6382, "guardiantowerb", "Slime");
+
+        //Get the Slime 6383
+        Story.KillQuest(6383, "guardiantowerb", "Slime");
+
+        //Reach for the Heart 6384
+        Story.KillQuest(6384, "guardiantowerb", "Yargol Magebane");
+
+        //Get More Shadows 6385
+        Story.KillQuest(6385, "guardiantowerb", "Slime");
+
+        //Garen's Key 6386
+        Story.KillQuest(6386, "guardiantowerb", new[] { "Guardian Selby", "Guardian Garen"});
+
+        //Selby's Key 6387
+        Story.KillQuest(6387, "guardiantowerb", new[] { "Guardian Garen", "Guardian Selby" });
+
+        //Get Rid of Bolton 6388
+        Story.MapItemQuest(6388, "guardiantowerb", 5901);
+        Story.KillQuest(6388, "guardiantowerb", "Guardian Bolton");
+        
+        //Get Rid of the Imps 6389
+        Story.KillQuest(6389, "ebondungeon", "Ebonslate Imp");
+
+        //Get Into the Dungeon 6390
+        Story.MapItemQuest(6390, "ebondungeon", 5902);
+
+        //Destroy the Guards 6391
+        Story.KillQuest(6391, "ebondungeon", "Ebon Dungeon Guard");
+
+        //Get the Key 6392
+        Story.KillQuest(6392, "ebondungeon", "Elite Dungeon Guard");
+
+        //Find Lynaria 6393
+        Story.MapItemQuest(6393, "ebondungeon", new[] { 5903, 5904 });
+
+        //Destroy Dethrix 6394
+        Story.KillQuest(6394, "ebondungeon", "Dethrix");
+
+        //Siphon the Power (Farming) 6395
+        Story.KillQuest(6395, "ebondungeon", "Ebon Dungeon Guard");
+
+        //He's Not Your Friend 6399
+        Story.KillQuest(6399, "alteonfight", "King Alteon");
+
+        //Defeat the Dragon 6400
+        Story.MapItemQuest(6400, "alteonfight", 5910);
+
+        //Power Boost 6401
+        Story.KillQuest(6401, "darkplane", "Shadow Beast");
+
+        //Take His Knights 6402
+        Story.KillQuest(6402, "darkplane", "Guardian Knight");
+
+        //Destroy the Light 6403
+        Story.KillQuest(6403, "darkplane", "Light Spirit");
+
+        //More Power! 6404
+        Story.KillQuest(6404, "darkplane", "Shadow Beast");
+
+        //Destroy the Force Field 6405
+        Story.MapItemQuest(6405, "darkplane", 5911);
+
+        //Defeat the Battalion 6406
+        Story.KillQuest(6406, "darkplane", "Guardian Knight");
+
+        //Destroy the Beast 6407
+        Story.KillQuest(6407, "darkplane", "Victorious");
+
+        //Learn from the Past 6408
+        Story.KillQuest(6408, "darkplane", "Shadow Beast");
+
```

Comment:
line spacing - something exe is very specific about ^_^ - each quest gets a extra line after, and end of files are to be closed without spaces ^ looks good otherwise -- with the fixes/questions mentioned